File: Iniciante/URI 1011.c
#include <stdio.h>
#include <math.h>

#define pi 3.14159 

int main() { 
    double raio, volume;

    scanf("%lf", &raio);
    volume = (4.0 / 3) * pi * pow(raio, 3.0);
    printf("VOLUME = %.3f\n", volume);

    return 0;
}

File: Iniciante/URI 1011.go
package main
 
import (
    "fmt"
    "math"
)

const pi = 3.14159

func main() {
    var raio, volume float64

    fmt.Scanf("%f", &raio)
    volume = (4.0 / 3) * pi * math.Pow(raio, 3.0)
    fmt.Printf("VOLUME = %.3f\n", volume)
}